test/helpers/index.coffee

```diff
@@ -0,0 +1,81 @@
+_ = require 'underscore'
+expect = chai.expect
+React = require 'react'
+ReactAddons    = require('react/addons')
+ReactTestUtils = React.addons.TestUtils
+{Promise}      = require 'es6-promise'
+{commonActions} = require './utilities'
+sandbox = null
+ROUTER = null
+CURRENT_ROUTER_PARAMS = null
+CURRENT_ROUTER_PATH   = null
+CURRENT_ROUTER_QUERY = null
+# Mock a router for the context
+beforeEach ->
+  sandbox = sinon.sandbox.create()
+  ROUTER  = sandbox.spy()
+  ROUTER.makeHref = sandbox.spy()
+  ROUTER.isActive = sandbox.spy()
+  ROUTER.transitionTo = sandbox.spy()
+  ROUTER.getCurrentPath = sandbox.spy( -> CURRENT_ROUTER_PATH )
+  ROUTER.getCurrentQuery = sandbox.spy( -> CURRENT_ROUTER_QUERY )
+  ROUTER.getLocation = sandbox.spy( ->
+    addChangeListener: sandbox.spy()
+    removeChangeListener: sandbox.spy()
+  )
+  ROUTER.match = sandbox.spy()
+  ROUTER.getCurrentParams = sandbox.spy( -> CURRENT_ROUTER_PARAMS )
```

Comment:
It seems odd to have the `router` stub logic in here since we don't use `react-router` at all in this repo.  I see that having that here could mean that we could use this `TestComponent` from this shared components repo, which makes sense...I wonder though, would it make more sense to pull the router related logic out into a separate file, and make a separate testing helper that would specifically for router dependent components?  Or, if we rename this helper, and use a simpler helper for non-router components -- i.e. all those in this repo.